Jenkins job is fails because it has NodeJS v12

### Summary

$sbj. https://github.com/ansible/vscode-ansible/pull/251 attempted to fix the pipeline by updating NPM. Now, it fails because of a lower NodeJS version that we declare in `package.json`.

### Extension version

`main`

### VS Code version

N/A

### Ansible Version

```shell
N/A
```


### OS / Environment

Jenkins, NodeJS v12.20.0

### Relevant log output

```shell
14:55:15  npm ERR! notsup Unsupported engine for ansible@0.5.0: wanted: {"node":">=16.0","npm":">=7.12.1","vscode":"^1.48.0","yarn":"\n\nERROR: Please use npm, yarn is not supported in this repository!!!\n\n"} (current: {"node":"12.20.0","npm":"6.14.8"})
```

--- a/Jenkinsfile
+++ b/Jenkinsfile
@@ -1,7 +1,7 @@
 #!/usr/bin/env groovy
 
 def installBuildRequirements(){
-  def nodeHome = tool 'nodejs-12.20.0'
+  def nodeHome = tool 'nodejs-16.12.0'
   env.PATH="${env.PATH}:${nodeHome}/bin"
 
   sh "npm install --global vsce@latest npm@latest"

